Makefile

```diff
@@ -24,17 +24,15 @@ smoke-test:
 	# Smoke tests run in parallel
 	pytest -n 4 tests/functional
 
-flake:
-	# Make sure code conforms to PEP8 standards
-	flake8 samcli
-	flake8 tests/unit tests/integration
-
 lint:
 	# Linter performs static analysis to catch latent bugs
 	pylint --rcfile .pylintrc samcli
 
 # Command to run everytime you make changes to verify everything works
-dev: flake lint test
+dev: lint test
+
+black:
+	black samcli/* tests/*
 
 # Verifications to run before sending a pull request
-pr: init dev
+pr: init dev
```

Comment:
Why isn't `black` run as part of `make pr`?

Reply:
the `make black` command requires you to commit the changes. So I didn't want the `make pr` to run that and make it look like it works. I can add another `make` command that does the check instead, which is probably what we want anyways.

Reply:
If you install the git hooks, running the check on pr is just redundant. This is because you can't push unless black passes.